tests/integrations/config/dynamic/gc_worker.rs

```diff
@@ -0,0 +1,155 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::f64::INFINITY;
+use std::sync::mpsc::channel;
+use std::time::Duration;
+use tikv::config::{ConfigController, TiKvConfig};
+use tikv::server::gc_worker::GcConfig;
+use tikv::server::gc_worker::{GcTask, GcWorker};
+use tikv::storage::kv::TestEngineBuilder;
+use tikv_util::config::ReadableSize;
+use tikv_util::time::Limiter;
+use tikv_util::worker::FutureScheduler;
+
+#[test]
+fn test_gc_config_validate() {
+    let cfg = GcConfig::default();
+    cfg.validate().unwrap();
+
+    let mut invalid_cfg = GcConfig::default();
+    invalid_cfg.batch_keys = 0;
+    assert!(invalid_cfg.validate().is_err());
+}
+
+fn setup_cfg_controller(
+    cfg: TiKvConfig,
+) -> (GcWorker<tikv::storage::kv::RocksEngine>, ConfigController) {
+    let engine = TestEngineBuilder::new().build().unwrap();
+    let mut gc_worker = GcWorker::new(engine, None, None, None, cfg.gc.clone());
+    gc_worker.start().unwrap();
+
+    let mut cfg_controller = ConfigController::new(cfg, Default::default());
+    cfg_controller.register("gc", Box::new(gc_worker.get_config_manager()));
+
+    (gc_worker, cfg_controller)
+}
+
+fn validate<F>(scheduler: &FutureScheduler<GcTask>, f: F)
+where
+    F: FnOnce(&GcConfig, &Limiter) + Send + 'static,
+{
+    let (tx, rx) = channel();
+    scheduler
+        .schedule(GcTask::Validate(Box::new(
+            move |cfg: &GcConfig, limiter: &Limiter| {
+                f(cfg, limiter);
+                tx.send(()).unwrap();
+            },
+        )))
+        .unwrap();
+    rx.recv_timeout(Duration::from_secs(3)).unwrap();
+}
+
+#[allow(clippy::float_cmp)]
+#[test]
+fn test_gc_worker_config_update() {
```

Comment:
Please also remove the coresponding tests in `gc_worker.rs`

Reply:
Fixed.